src/main/java/duke/helper/TaskList.java

```diff
@@ -0,0 +1,76 @@
+package duke.helper;
+
+import duke.task.Task;
+
+import java.util.ArrayList;
+
+public class TaskList {
+    private ArrayList<Task> tasks = new ArrayList<>();
+
+    public ArrayList<Task> getTasks() {
+        return tasks;
+    }
+
+    /**
+     * Adds a task to the tasklist.
+     *
+     * @param currTask the task to be added.
+     * @return The string to be printed
+     */
+    public String addTask(Task currTask) {
+        tasks.add(currTask);
+        return String.format("Added task nya!\n  %s\n"
+                + " Nyow you have %d tasks in the list nya!\n", currTask, tasks.size());
+    }
+
+    /**
+     * Deletes a task from the tasklist.
+     *
+     * @param id, the index of task in the tasklist to delete.
+     * @return The string to be printed.
+     */
+    public String deleteTask(int id) {
+        return String.format("Removed task nya!\n  %s\n" +
+                        " Nyow you have %d tasks in the list nya!\n",
+                tasks.remove(id - 1), tasks.size());
+
+    }
+
+    /**
+     * Marks a task from the tasklist.
+     *
+     * @param id, the index of task in the tasklist to mark.
+     * @return The string to be printed.
+     */
+    public String markTask(int id) {
+        Task currTask = tasks.get(id - 1);
+        currTask.setDone();
+        return String.format("Good job for doing your task nya!\n  %s\n", currTask);
+    }
+
```

Comment:
I like that you have used javadocs to explain some of your `classes` and `public` methods! 😊
Adding on to the other reviewer's comment, perhaps you could add it to all the other methods **except** `getters/setters` and `overridden methods` (given that the parent method has javadoc).